submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md

```diff
@@ -0,0 +1,61 @@
+# OpenFF multi-Br ESP Fragment Conformers v1.0
+
+## Description
+
+A dataset containing fragments of molecules from the Zinc, enamine, and CHEMBL datasets, optimized at the B3LYP-D3BJ/DZVP level of theory. Molecules containing I were fragmented into small fragments. 
```

Comment:
```suggestion
A dataset containing fragments of molecules from the Zinc, Enamine 10240, Enamine 50240, and CHEMBL datasets, optimized at the B3LYP-D3BJ/DZVP level of theory. Molecules containing I were fragmented into small fragments. 
```

(just for full clarity -- there are quite a few enamine datasets!)

Reply:
I think this description is in the dataset as generated by the notebook, as well.